tests/integration_test/services_test.py

```
@@ -23655,7 +23698,7 @@
     assert actual["error_code"] == 0
```

Ruff (0.12.2)

Use of `assert` detected

(S101)